test/e2e/performanceprofile/functests/README.txt

```diff
@@ -1,5 +1,16 @@
-HEADS UP!
+Functest consist in diffent suites:
 
-We have 2 test suites here, one running the "normal" performance tests, the other one performance profile update tests.
-The latter should run AFTER the former, and tests are executed in order of filenames.
-So be careful with renaming existing or adding new suites.
\ No newline at end of file
+0_config should be run before other in order to deploy PerformanceProfile and wait for MCP to pick MC and begin in updated state
```

Comment:
maybe make a bulleted list here?

Reply:
also maybe keep the suites name consistent for the rest of the suites, for example either all like X_<suite> (0_config) or <suite name> (e.g Configuration suite) while mentioning the actual suite folder name